### PR TITLE
Added information about OpenVINO toolkit to login page

### PR DESCRIPTION
**Pull request by azhavoro**
### Motivation and context
image

### Checklist

- [x] I submit my changes into the `develop` branch
- [x] I have added a description of my changes into [CHANGELOG](https://github.com/opencv/cvat/blob/develop/CHANGELOG.md) file
~~- [ ] I have updated the [documentation](
  https://github.com/opencv/cvat/blob/develop/README.md#documentation) accordingly~~
~~- [ ] I have added tests to cover my changes~~
~~- [ ] I have linked related issues ([read github docs](
  https://help.github.com/en/github/managing-your-work-on-github/linking-a-pull-request-to-an-issue#linking-a-pull-request-to-an-issue-using-a-keyword))~~
- [x] I have increased versions of npm packages if it is necessary ([cvat-canvas](https://github.com/opencv/cvat/tree/develop/cvat-canvas#versioning),
[cvat-core](https://github.com/opencv/cvat/tree/develop/cvat-core#versioning), [cvat-data](https://github.com/opencv/cvat/tree/develop/cvat-data#versioning) and [cvat-ui](https://github.com/opencv/cvat/tree/develop/cvat-ui#versioning))

### License

- [x] I submit _my code changes_ under the same [MIT License](
  https://github.com/opencv/cvat/blob/develop/LICENSE) that covers the project.
  Feel free to contact the maintainers if that's a concern.
- [x] I have updated the license header for each file (see an example below)

```python
# Copyright (C) 2021 Intel Corporation
#
# SPDX-License-Identifier: MIT
```

### CODE REVIEW

**Comment by nmanovic** [comment]
@azhavoro , could you please name the PR properly and add a screenshot?

**Review by bsekachev — DISMISSED** [review]

**Comment by bsekachev** [comment]
@Marishka17 Could you please review?

File: cvat-ui/src/assets/openvino.svg [hunk 1]
```diff
@@ -0,0 +1,14 @@
+<svg width="120" height="28" viewBox="0 -3 120 28" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (Marishka17) [inline cvat-ui/src/assets/openvino.svg:1]: @azhavoro or @bsekachev, Could you also add a license, please?

**Comment by nmanovic** [comment]
@yury-gorbachev , @kchechil , could you please look at the screenshot? Please let us know if you want to change wording.

File: cvat-ui/src/consts.ts [hunk 2]
```diff
@@ -25,6 +25,7 @@ const DEFAULT_PROJECT_SUBSETS = ['Train', 'Test', 'Validation'];
 const INTEL_TERMS_OF_USE_URL = 'https://www.intel.com/content/www/us/en/legal/terms-of-use.html';
 const INTEL_COOKIES_URL = 'https://www.intel.com/content/www/us/en/privacy/intel-cookie-notice.html';
 const INTEL_PRIVACY_URL = 'https://www.intel.com/content/www/us/en/privacy/intel-privacy-notice.html';
+const OPENVINO_URL = 'https://docs.openvino.ai/latest/index.html';
```
Comment (nmanovic) [inline cvat-ui/src/consts.ts:28]:
```suggestion
const OPENVINO_URL = 'https://docs.openvino.ai';
```

**Review by nmanovic — APPROVED** [review]
LGTM, updated the openvinotoolkit link.

**Pull request merged**